src/vmm/tests/integration_tests.rs

```diff
@@ -178,3 +184,60 @@ fn test_pause_resume_microvm() {
         }
     }
 }
+
+#[test]
+#[cfg(target_arch = "x86_64")]
+fn test_create_snapshot() {
+    let snapshot_path = PathBuf::from("snapshot_file");
+    let mem_file_path = PathBuf::from("mem_file");
+
+    let pid = unsafe { libc::fork() };
+    match pid {
+        0 => {
+            set_panic_hook();
+            let boot_source_cfg: BootSourceConfig =
+                MockBootSourceConfig::new().with_default_boot_args().into();
+            let resources: VmResources = MockVmResources::new()
+                .with_boot_source(boot_source_cfg)
+                .into();
+            let mut event_manager = EventManager::new().unwrap();
+            let empty_seccomp_filter = get_seccomp_filter(SeccompLevel::None).unwrap();
+
+            let vmm = build_microvm(&resources, &mut event_manager, &empty_seccomp_filter).unwrap();
+
+            // Give the vCPUs a chance to attempt KVM_RUN.
```

Comment:
This looks imported from the seccomp test, where the vcpus fail at KVM_RUN, please update the comment so that it doesn't hint at bad things happening

Reply:
Done.